### DISCUSSION TITLE
Make action modal more "widenable"

### DISCUSSION THREAD

**Discussion by kusab85** [laravel/nova-issues#4809] [Ideas & Feature Requests] [answered]
It'a all great @davidhemphill  that you've added ``size`` feature to ``Action`` but...
...why there are **so few** size options ???

**Modal.vue**
```javascript
 size: {
      type: String,
      default: 'xl',
      validator: v => ['sm', 'md', 'lg', 'xl', '2xl'].includes(v),
    },
```

Please add more of https://tailwindcss.com/docs/max-width

```
3xl, 4xl, 5xl, 6xl, 7xl, screen-sm, screen-md, screen-lg, screen-xl, screen-2xl
```

**Comment by davidhemphill** [comment, marked as answer]
I've added more modal sizes, up to '7xl'. Past that, you should probably just set the modal to fullscreen instead. The new sizes will come in the next release.

**Reply by kusab85** [reply]
Thank you very much David!
